style.css

```diff
@@ -1,3 +1,7 @@
 body{
   backround-color:red
 }
+#title{
```

Comment:
make sure you apply correct spacing in your css to make your code easy to read and go through

Reply:
> make sure you apply correct spacing in your css to make your code easy to read and go through

- [ ] I would advise you to recommend changes politely, and not with commands. I think it would be better written like 'kindly apply spacing in your...'